tests/integration/test_cli.py

```diff
@@ -47,6 +51,7 @@ def test_worker(entrypoint, click_app, mocker):
         listen_notify=False,
         delete_jobs="always",
     )
+    log.assert_called_once_with(verbosity=3, format="{message},yay!", style="{")
```

Comment:
Don't you think that it would be good to have a real integration test where we can check that logs from Procrastinate have the expected user-defined format?

Reply:
ok